ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -113,6 +113,7 @@ public async Task correctlyCreates_basicNotification()
                 await _importValidator.CleanAndValidateNotification(null, runId, notification);
 
             // Assert
+            Assert.Equal("ETS", notification.LegacySource);
```

Comment:
We should add a test somewhere checking that the `LegacyId` on a notification is being set correctly. That would be a test of the notification model, so may require a new test class.

Reply:
Have added a new test class for this